Update minimizable sample to include documentation on reconnecting (#3239)
microsoft/BotFramework-WebChat: samples/06.recomposing-ui/a.minimizable-web-chat/README.md

First we will render Web Chat. To test, you can temporarily add `<WebChat>` to your `App.js`

In the `WebChat.js` file, import `React`, `{ useEffect, useMemo }` `ReactWebChat`, `createDirectLine`, and `createStyleSet` from our packages.

<!-- prettier-ignore-start -->
```js
import React from 'react';
import ReactWebChat, { createDirectLine, createStyleSet } from 'botframework-webchat';
```
<!-- prettier-ignore-end -->

> Set up the function component the same way you would set up a regular `React.Component`

Reviewer: ```suggestion
Set up the functional component the same way you would set up a regular `React.Component`
```

1. Set state with `useMemo` and `createDirectLine` with your token. It is **never recommended** to put the Direct Line secret in the browser or client app. To learn more about secrets and tokens for Direct Line, visit this [tutorial on authentication](https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-direct-line-3-0-authentication).
1. Inside render, be sure to return the `<ReactWebChat>` component.

<!-- prettier-ignore-start -->
```js
const WebChat = ({ className, onFetchToken, store, token }) => {
  const directLine = useMemo(() => createDirectLine({ token }), [token]);

  useEffect(() => {
    onFetchToken();
  }, [onFetchToken]);

  return token ? (
    <ReactWebChat className={`${className || ''} web-chat`} directLine={directLine} store={store} styleSet={styleSet} />
  ) : (
    <div className={`${className || ''} connect-spinner`}>
      <div className="content">
        <div className="icon">
          <span className="ms-Icon ms-Icon--Robot" />
        </div>
        <p>Please wait while we are connecting.</p>
      </div>
    </div>
  );
};
```
<!-- prettier-ignore-end -->

> With `useEffect`, invoke `onFetchToken` from props if the `token` has not been passed from the parent component.

Reviewer: ```suggestion
In the `useEffect` hook, invoke `onFetchToken` from props if the `token` has not been passed from the parent component.
```

Reviewer: This isn't really clear because the logic to check if the token has been fetched is in the `onFetchToken` - not the in the `useEffect` hook.

<!-- prettier-ignore-start -->
```js
  useEffect(() => {
    onFetchToken();
  }, [onFetchToken]);

```
<!-- prettier-ignore-end -->

> Note: `onFetchToken()` is a method that will be implemented in `MinimizableWebChat.js`.

Let's move on to building the `<MinimizableWebChat>` component.

Import `React`, `{ useCallback, useMemo, useState }` `createStore`, and `createStyleSet`. Then import your newly made component, `WebChat`.

<!-- prettier-ignore-start -->
```js
import React from 'react', { useCallback, useMemo, useState };
import { createStore, createStyleSet } from 'botframework-webchat';

import WebChat from './WebChat';
```
<!-- prettier-ignore-end -->

> 11. Build the component in the return statement

Reviewer: ```suggestion
1. Build the component in the return statement
```
